README.md

```diff
@@ -29,26 +29,34 @@ source ./.env/Scripts/activate
 pip install -r requirements.txt
 ```
 
+### **Create project entry in the database**:
+
+Make sure that the project from which you collect data is already in the 
+project collection of the MongoDB. If not, you can add them by:
+
+```
+db.project.insert({"name": <PROJECT_NAME>})
+```
 ### **How to Run**:
 
 Simple quick run for a local MongoDB:
 <br />
 
 ```
-python main.py -ru [repository url]
+python main.py -ru [repository url] -pn [project name]
```

Comment:
This change needs to be reflected in: plugin_packaging/info.json and plugin_packagin/execute.sh

It already has the repository it just needs to be required and renamed in both.